state/indexer/block/kv/kv.go

```diff
@@ -23,7 +24,11 @@ import (
 	"github.com/cometbft/cometbft/types"
 )
 
-var _ indexer.BlockIndexer = (*BlockerIndexer)(nil)
```

Comment:
I would leave this line: `var _ indexer.BlockIndexer = (*BlockerIndexer)(nil)`. It produces an easier-to-understand error if the type doesn't implement the interface